src/index.js

```diff
@@ -48,11 +40,10 @@ export default function(deckName) {
 
   models[top_model_id + ''] = model;
 
-  update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
+  exporter.update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
 
-  return {
-    addMedia,
-    addCard,
-    save
-  };
+  return ['addCard', 'addMedia', 'save'].reduce((prev, i) => {
```

Comment:
No need for that, adding more functions doesn't break anything. You can apply it to anything which doesn't make sense to you. I will release it as a breaking change `v3.0.0` (as too much of the code are rewritten) when any new features are added.